frontend/components/MultiColumnContent.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <div :style="getStyle">
+    <div v-for="(section, index) in sections" :key="index">
+      <div class="inline-block mt-6">
+        <h3 class="text-xxl italic mb-0">
+          <a :href="section.link">
+            {{ section.title }}
+          </a>
+        </h3>
+        <ul>
+          <li
+            v-for="(subItem, index) in section.subItems"
+            :key="index"
+            class="mt-2 first:mt-3"
+          >
+            <a :href="subItem.link" class="text-blue hover:text-blueh">
+              {{ subItem.text }}
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    sections: {
+      type: Array,
+      default: () => [],
+    },
+    columnVariant: {
```

Comment:
@codeabilly  I understand this might not be your preferred approach. However, being explicit about the variant we use helps other developers. Instead of relying on unclear logic to determine the needed columns, I'd prioritize clarity over extreme flexibility